package.json

```diff
@@ -31,6 +31,7 @@
     "@govuk-one-login/di-ipv-cri-common-express": "6.3.0",
     "@govuk-one-login/frontend-analytics": "1.0.3",
     "@govuk-one-login/frontend-language-toggle": "1.1.0",
+    "@govuk-one-login/frontend-passthrough-headers": "1.0.0",
```

Comment:
We'll also need to upgrade common express to 6.4.0 so that we have your changes from this PR https://github.com/govuk-one-login/ipv-cri-common-express/pull/366

Reply:
updated! Thanks!